### PR TITLE
Add image for ads

### PR DESCRIPTION
**Pull request by adeelnoor**
Temporary solution of uploading the images for an ad

### CODE REVIEW

File: Controllers/AdsImageController.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+
+
+const prisma = new PrismaClient({
+  errorFormat: 'minimal',
+})
+
+async function uploadImageToImgbb(imageBuffer) {
+    try {
+        const apiKey = '51f315839f24916a3af27e87d3d05c3c'; // Replace with your ImgBB API key
```
Comment (Mutahir2016) [inline Controllers/AdsImageController.js:6]: key to goto .env file...

File: Controllers/AdsImageController.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        const imageBase64 = imageBuffer.toString('base64');
+
+        const formData = new URLSearchParams();
```
Comment (Mutahir2016) [inline Controllers/AdsImageController.js:9]: can we use `new FormData();` instead of `URLSearchParams()` , otherwise existing is fine

**Review by Mutahir2016 — APPROVED** [review]
Looks good 👍

File: Controllers/AdsImageController.js (hunk 2 above)
Comment (adeelnoor) [reply Controllers/AdsImageController.js:9]: `URLSearchParams` is native node.js api method

**Pull request merged**